modules/ROOT/pages/push/proc_preparing-ups-for-use-with-ios.adoc

```diff
@@ -0,0 +1,109 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: proc_preparing-ups-for-use-with-ios
+
+[id='{context}_preparing-ups-for-use-with-ios']
+= Preparing {unifiedpush-service} For Use With iOS
+
+Before using {unifiedpush-service} with iOS, you must complete the following actions:
+
+* Request a Signed Certificate
+* Create an Apple App ID and SSL Certificate for APNs
```

Comment:
What does "APNs" stand for?

Reply:
Apple Push Notification service. They also use a lowercase s in their own docs: https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1

Reply:
I've created a file ' modules/ROOT/pages/push/ref_terminology.adoc' to cater for this (and any additional push related terminology that we feel should be documented)